### ISSUE TITLE
2.4.3 Focus Order (What's Next to be Loaded?)

### ISSUE THREAD

**Issue by AlanaBayless** [UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan#42]
**Were violation(s) found?**
(If Yes, describe in the comments.)
- [ ] No violations found
- [ ] Yes, violations found and all fixed
- [ ] Yes, violations found and some fixed while others are unresolved _(Add **6.Unresolved 508** label)_
- [ ] Yes, violations found but remain unresolved _(Add **6.Unresolved 508** label)_
- [ ] Flag as a potential problem _(Add **4.Help** label)_

---
### 2.4.3 Focus Order - Level A
If a Web page can be navigated sequentially and the navigation sequences affect meaning or operation, focusable components receive focus in an order that preserves meaning and operability.

[Understanding 2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)
[How to Meet 2.4.3](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=131%2C133%2C141%2C143%2C211%2C212%2C221%2C231%2C242%2C243#focus-order)
[How to Test 2.4.3](https://usfs.box.com/s/knn5w8jul8g0xicrnpa4cxzk07t8rc1d)

**Trusted Tester IDs:**
- [x] Test 4.F focus-order meaning
- [x] Test 4.G focus-order-reveal
- [x] Test 4.H focus-order-return

**Accessibility Principle:** Operable (Navigable)

**Comment by AlanaBayless** [comment]
The Loaded Data page pass 2.4.3 Focus Order, see Trusted Tester IDs below:

**Trusted Tester IDs:**

- [ ]  Test 4.F focus-order meaning: PASS, The tab order of the page is in the correct order, which preserves the meaning of the page.
- [ ]  Test 4.G focus-order-reveal: DNA, there are no features that reveals content on the page.
- [ ]  Test 4.H focus-order-return: DNA, there are no features that reveals content on the page.

**Comment by AlanaBayless** [comment]
Since this page pass all test IDs associated with 2.4.3 Focus order, this page is QA Certified.
